src/screens/Home/components/HomeMenuItem.tsx

```diff
@@ -11,20 +16,21 @@ type sublabel = {
 
 interface HomeMenuItemProps {
   label: string;
-  amount?: number;
+  amount: number;
   sublabels?: sublabel[];
   noBadge?: boolean;
-  classes: { root: string; content: string; noAlert: string; sublabel: string; icon: string; grey: string };
+  iconType?: string;
+  classes: { root: string; content: string; noAlert: string; badgeStyles: string; checked: string; innerContent: string; };
 }
 
 const styles = (theme: Theme) =>
   createStyles({
     root: {
       width: '100%',
       border: '1px solid',
-      borderColor: theme.palette.divider,
+      borderColor: theme.palette.primary.main,
       borderRadius: '6px',
-      padding: '15px 15px',
+      padding: 15,
```

Comment:
Nit: where possible, try to replace custom px spacings with `theme.spacing` to keep it consistent with an 8pt grid.